event-handlers/src/main/java/no/sikt/nva/nvi/events/batch/model/StartBatchJobRequest.java

```diff
@@ -0,0 +1,47 @@
+package no.sikt.nva.nvi.events.batch.model;
+
+import static java.util.Collections.emptyList;
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static java.util.Objects.requireNonNull;
+import static java.util.Objects.requireNonNullElse;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import no.unit.nva.commons.json.JsonSerializable;
+
+public record StartBatchJobRequest(
+    BatchJobType jobType,
+    ReportingYearFilter filter,
```

Comment:
Maybe name ReportingYearFilter something more generic so that the next person extends this instead of createing filter2 parameter